drivers/wifi/nrf700x/src/net_if.c

```diff
@@ -169,14 +171,29 @@ int nrf_wifi_if_send(const struct device *dev,
 		goto out;
 	}
 
-	if ((vif_ctx_zep->if_carr_state != NRF_WIFI_FMAC_IF_CARR_STATE_ON) ||
-	    (!vif_ctx_zep->authorized && !is_eapol(pkt))) {
-		goto out;
-	}
+#ifdef CONFIG_NRF700X_RAW_DATA_TX
+	memcpy(&magic_num, (unsigned int *)pkt->frags->data, sizeof(unsigned int));
+	if (magic_num == NRF_WIFI_MAGIC_NUM_RAWTX) {
```

Comment:
Can just compare with `pkt->frags->data` directly and get rid of `magic_num` variable and extra `memcpy`

Reply:
The code is cleaner and it does not save a lot but okay

Reply:
done